src/scss/coral-talk-iframe/main.scss

```diff
@@ -327,3 +339,32 @@ a {
 		color: oColorsGetPaletteColor(oColorsGetUseCase(link-hover, text)) !important;
 	}
 }
+
+
+// make font sizes consistent in various places
+.coral-guidelines div div,
+.coral-rte-content,
+.coral-rte-placeholder,
+.coral-comment-content,
+.coral-myComment-content,
+.coral-myComment-story,
+.coral-createReplyComment-replyToText,
+.coral-viewerBox div div,
+.coral-viewerBox div div span,
+.coral-viewerBox div div button,
+.coral-editComment div div div div span,
+.coral-editComment-expiredTime,
+.coral-editComment-remainingTime span,
+.coral-sortMenu label,
+.coral-sortMenu span select,
+.coral-tabBarSecondary-tab div span,
+.coral-moderationDropdown-featureButton div div,
+.coral-moderationDropdown-unfeatureButton div div,
+.coral-moderationDropdown-approveButton div div,
+.coral-moderationDropdown-approveedButton div div, // typo in Coral class name
```

Comment:
I am a bit concerned about how specific some of these styles are, could cause us to be fixing bugs whenever a change is made to Coral UI. 

Wondering how close we can get to our goal without being so specific.